README.md

```diff
@@ -36,7 +36,7 @@ This package provides various widgets for `teal` applications. Examples include:
 
 ```r
 # stable versions
-install.packages('teal.widgets', repos = c('https://insightsengineering.r-universe.dev', getOption('repos')))
+install.packages('teal.widgets')
```

Comment:
This is fine for me. But, I was wondering if it's a good idea to also add a way to install a stable version from the r-universe?
The only downside that I see is that r-universe does not great with `renv` so I'm leaning more towards not including that.

Reply:
I think we prefer CRAN over r-universe because CRAN is well-known. Furthermore, all the other packages we have released only include CRAN for the stable version. For the sake of consistency, let's continue to install from CRAN.